.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Gitter chat
-    url: https://gitter.im/tldr-pages/tldr
+  - name: Chatroom
+    url: https://matrix.to/#/#tldr-pages:matrix.org
     about: You can ask and answer questions here
   - name: Matrix Space
-    url: https://matrix.to/#/#tldr-pages:matrix.org
-    about: Matrix view of our Gitter chat rooms
+    url: https://matrix.to/#/#tldr-pages-project:matrix.org
+    about: Matrix space of our chat rooms
```

Comment:
I'm wondering if we should avoid referring people to the space, as that's a slightly confusing aspect of Matrix still. Maybe we should just drop this link. WDYT?

Reply:
Yeah, that seems better.

Reply:
If we want to allow others to chime in, we shouldn't mark the conversation as resolved :)

Reply:
I personally don't see the issue with spaces myself.
It's similar to workspaces on Slack, or servers/guilds on Discord. A fairly well-known concept.

While Matrix does this a bit differently from other platforms, the surface-level UX should be familiar.

However, in the end it makes little difference while we don't have many rooms I guess. So whatever works. :sweat_smile:

Reply:
> So whatever works. 😅

Yeah